pkg/recipes/driver/bicep_test.go

```diff
@@ -371,7 +372,7 @@ func Test_Bicep_Execute_SimulatedEnvironment(t *testing.T) {
 			Definition: recipes.EnvironmentDefinition{
 				Name:         "test-recipe",
 				Driver:       recipes.TemplateKindBicep,
```

Comment:
This test doesn't really care what the path value is. I will create a separate PR to fix this test to use a random test template path